Compile issue ROS Indigo

Hi, I try to build the hector_camera_control package using ros indigo, ubuntu 14.04. i cloned also the necessary messages and get the following error:
`In file included from /opt/ros/indigo/include/ros/serialization.h:37:0,
                 from /opt/ros/indigo/include/ros/publisher.h:34,
                 from /opt/ros/indigo/include/ros/node_handle.h:32,
                 from /opt/ros/indigo/include/ros/ros.h:45,
                 from /home/naegelit/catkin_ws/src/camera_pacacement_ros/hector_quadrotor/hector_camera_control/hector_camera_joint_controller/include/hector_camera_joint_controller/cam_joint_trajectory_controller.h:37,
                 from /home/naegelit/catkin_ws/src/camera_pacacement_ros/hector_quadrotor/hector_camera_control/hector_camera_joint_controller/src/cam_joint_trajectory_controller.cpp:29:
/opt/ros/indigo/include/ros/message_traits.h: In instantiation of ‘static const char* ros::message_traits::MD5Sum<M>::value(const M&) [with M = double]’:
/opt/ros/indigo/include/ros/message_traits.h:255:104:   required from ‘const char* ros::message_traits::md5sum(const M&) [with M = double]’
/opt/ros/indigo/include/ros/publisher.h:112:7:   required from ‘void ros::Publisher::publish(const M&) const [with M = double]’
/home/naegelit/catkin_ws/src/camera_pacacement_ros/hector_quadrotor/hector_camera_control/hector_camera_joint_controller/src/cam_joint_trajectory_controller.cpp:364:37:   required from here
/opt/ros/indigo/include/ros/message_traits.h:126:34: error: request for member ‘__getMD5Sum’ in ‘m’, which is of non-class type ‘const double’
     return m.__getMD5Sum().c_str();
                                  ^
/opt/ros/indigo/include/ros/message_traits.h: In instantiation of ‘static const char* ros::message_traits::DataType<M>::value(const M&) [with M = double]’:
/opt/ros/indigo/include/ros/message_traits.h:264:106:   required from ‘const char* ros::message_traits::datatype(const M&) [with M = double]’
/opt/ros/indigo/include/ros/publisher.h:112:7:   required from ‘void ros::Publisher::publish(const M&) const [with M = double]’
/home/naegelit/catkin_ws/src/camera_pacacement_ros/hector_quadrotor/hector_camera_control/hector_camera_joint_controller/src/cam_joint_trajectory_controller.cpp:364:37:   required from here
/opt/ros/indigo/include/ros/message_traits.h:143:36: error: request for member ‘__getDataType’ in ‘m’, which is of non-class type ‘const double’
     return m.__getDataType().c_str();
                                    ^
make[3]: warning: jobserver unavailable: using -j1.  Add `+' to parent make rule.
Done building all
[ 99%] Built target ARSDKBuildUtils
[ 99%] Built target libbebop
[ 99%] Built target bebop_driver_node
[100%] Built target bebop_driver_nodelet
make[2]: *** [camera_pacacement_ros/hector_quadrotor/hector_camera_control/hector_camera_joint_controller/CMakeFiles/cam_joint_trajectory_controller.dir/src/cam_joint_trajectory_controller.cpp.o] Error 1
make[1]: *** [camera_pacacement_ros/hector_quadrotor/hector_camera_control/hector_camera_joint_controller/CMakeFiles/cam_joint_trajectory_controller.dir/all] Error 2
make: *** [all] Error 2
Invoking "make -j8 -l8" failed`




can you reproduce this? 

best,
Tobi

--- a/hector_camera_joint_controller/src/cam_joint_trajectory_controller.cpp
+++ b/hector_camera_joint_controller/src/cam_joint_trajectory_controller.cpp
@@ -31,6 +31,7 @@
 #include <control_msgs/FollowJointTrajectoryAction.h>
 #include <angles/angles.h>
 #include <std_msgs/Float64.h>
+#include <std_msgs/builtin_double.h>
 
 namespace cam_control {
 

